accumulo-store/src/test/java/gaffer/accumulostore/key/core/impl/ColumnQualifierVisibilityValueAggregatorTest.java

```diff
@@ -101,19 +105,32 @@ private void testAggregatingMultiplePropertySets(final MockAccumuloStore store,
             edge.setDestination("2");
```

Comment:
Just for future reference, these tests don't need changing, we now have builders for edges and entities. So you can do the following:

```java
final Edge expectedEdge = new Edge.Builder()
    .source("1").dest("2").directed(true)
    .property(AccumuloPropertyNames.COLUMN_QUALIFIER, 9)
    .property(AccumuloPropertyNames.PROP_1, 0)
    .property(AccumuloPropertyNames.PROP_2, 0)
    .property(AccumuloPropertyNames.PROP_3, 0)
    .property(AccumuloPropertyNames.PROP_4, 0)
    .property(AccumuloPropertyNames.COUNT, 15)
    .build();
```